examples/webots/vacuum/vacuum_simple.scenic

```diff
@@ -0,0 +1,146 @@
+param duration = 10
+
+## Class Definitions ##
+
+class Vacuum(WebotsObject):
+    webotsName: "IROBOT_CREATE"
+    shape: CylinderShape()
+    width: 0.335
+    length: 0.335
+    height: 0.07
+
+# Floor uses builtin Webots floor to keep Vacuum Sensors from breaking
+# Not actually linked to WebotsObject because Webots floor is 2D
+class Floor(Object):
+    width: 5
+    length: 5
+    height: 0.01
+    position: (0,0,-0.005)
+    color: [200, 200, 200]
+
+class Wall(WebotsObject):
+    webotsAdhoc: {'physics': False}
+    width: 5
+    length: 0.04
+    height: 0.5
+    color: [160, 160, 160]
+
+class DiningTable(WebotsObject):
+    webotsAdhoc: {'physics': True}
+    shape: MeshShape.fromFile(Path(localPath(".")).parent.parent.parent / "tools" / "meshes" / "dining_table.obj.bz2")
+    width: Range(0.7, 1.5)
+    length: Range(0.7, 1.5)
+    height: 0.75
+    density: 670 # Density of solid birch
+    color: [103, 71, 54]
+
+class DiningChair(WebotsObject):
+    webotsAdhoc: {'physics': True}
+    shape: MeshShape.fromFile(Path(localPath(".")).parent.parent.parent / "tools" / "meshes" / "dining_chair.obj.bz2", initial_rotation=(180 deg, 0, 0))
+    width: 0.4
+    length: 0.4
+    height: 1
+    density: 670 # Density of solid birch
+    positionStdDev: (0.05, 0.05 ,0)
+    orientationStdDev: (10 deg, 0, 0)
+    color: [103, 71, 54]
+
+class Couch(WebotsObject):
+    webotsAdhoc: {'physics': False}
+    shape: MeshShape.fromFile(Path(localPath(".")).parent.parent.parent / "tools" / "meshes" / "couch.obj.bz2", initial_rotation=(-90 deg, 0, 0))
+    width: 2
+    length: 0.75
+    height: 0.75
+    positionStdDev: (0.05, 0.5 ,0)
+    orientationStdDev: (5 deg, 0, 0)
+    color: [51, 51, 255]
+
+class CoffeeTable(WebotsObject):
+    webotsAdhoc: {'physics': False}
+    shape: MeshShape.fromFile(Path(localPath(".")).parent.parent.parent / "tools" / "meshes" / "coffee_table.obj.bz2")
+    width: 1.5
+    length: 0.5
+    height: 0.4
+    positionStdDev: (0.05, 0.05 ,0)
+    orientationStdDev: (5 deg, 0, 0)
+    color: [103, 71, 54]
+
+class Toy(WebotsObject):
+    webotsAdhoc: {'physics': True}
+    shape: Uniform(BoxShape(), CylinderShape(), ConeShape(), SpheroidShape())
+    width: 0.1
+    length: 0.1
+    height: 0.1
+    density: 100
+    color: [255, 128, 0]
+
+class BlockToy(Toy):
+    shape: BoxShape()
+
+## Scene Layout ##
+
+# Create room region and set it as the workspace
+room_region = RectangularRegion(0 @ 0, 0, 5.09, 5.09)
+workspace = Workspace(room_region)
+
+# Create floor and walls
+floor = new Floor
+wall_offset = floor.width/2 + 0.04/2 + 1e-4
+right_wall = new Wall at (wall_offset, 0, 0.25), facing toward floor
+left_wall = new Wall at (-wall_offset, 0, 0.25), facing toward floor
+front_wall = new Wall at (0, wall_offset, 0.25), facing toward floor
+back_wall = new Wall at (0, -wall_offset, 0.25), facing toward floor
+
+# Place vacuum on floor
+ego = new Vacuum on floor,
+    with customData str(random.getrandbits(32))
```

Comment:
Would be good to put a comment explaining what this is doing. Also it might make sense to move it to the `Vacuum` class.